### DISCUSSION TITLE
Trouble getting the Access Token

### DISCUSSION THREAD

**Discussion by Milkeeway** [potmat/homebridge-google-nest-sdm#151] [Q&A]
I followed every step and got up to the part where you need to put in the client id, client secret, and code,

When I put those in and copy the command into a terminal I get an error message saying: 

"Invoke-WebRequest : A parameter cannot be found that matches parameter name 'L'.
At line:1 char:6"

For reference I am on a Windows Machine. I had no luck with the command on Mac either, but it returned a different error. I prefer to use Windows when possible.

**Comment by gm268** [comment]
I had this today! In the end I used the terminal in Homebridge itself and it worked - though NB you need to double check you have all the id/secret/etc. 100% correct.

**Reply by bigbwb** [reply]
can you elaborate on how to use the terminal in Homebridge (I have the HOOBS box) and am new to it.  Just started adding plugins etc.  

Thanks!
